### DISCUSSION TITLE
Will v2 require PowerShell 7?

### DISCUSSION THREAD

**Discussion by joshtransient** [microsoftgraph/msgraph-sdk-powershell#1939] [General]
PnP and Graph SDK are my two most-used modules. PnP recently upgraded to .NET 6 for LTS reasons, and no longer works with Windows PowerShell 5.1. Wondering if Graph SDK v2 will also have that same fate.

**Comment by SeniorConsulting** [comment]
I'm currently using the V2.x on 5.1 at the moment, though it does have a bug (#1916) at the moment. It only happens sometimes. You can take it for a test drive yourself using:
Install-Module Microsoft.Graph -AllowPrerelease

Though do be aware that it's a preview module with more bugs than the generally available one.
